apps/web/lib/app-providers.tsx

```diff
@@ -75,7 +75,8 @@ const CustomI18nextProvider = (props: AppPropsWithoutNonce) => {
    **/
 
   const session = useSession();
-  const locale = session?.data?.user.locale ?? props.pageProps.newLocale;
+  const locale =
+    session?.data?.user.locale ?? typeof window !== "undefined" ? window.document.documentElement.lang : "en";
```

Comment:
why no use of props.pageProps.newLocale ?

Reply:
what's that?

Reply:
@Rksingh090 you removed "props.pageProps.newLocale" on L78 https://github.com/calcom/cal.com/blob/main/apps/web/lib/app-providers.tsx#L78